test/test_server.js

```diff
@@ -58,6 +58,12 @@ TestServer.start = function (config, printLogs) {
 }
 
 TestServer.prototype.start = function () {
+  if (this.config.lastAccessTimeUpdates.enabled) {
+    // HACK: Force-enable devices.lastAccessTime for test/remote/device_tests.js
+    process.env.LASTACCESSTIME_UPDATES_ENABLED = 'true'
+    process.env.LASTACCESSTIME_UPDATES_EMAIL_ADDRESSES = '.*'
+    process.env.LASTACCESSTIME_UPDATES_SAMPLE_RATE = '1'
```

Comment:
This was the hack I made to force config on the spawned server process. I'm happy to do it differently if this way offends anybody.